hongit-core/src/main/kotlin/com/devfloor/hongit/core/bookmarkboard/domain/BoardBookmark.kt

```diff
@@ -0,0 +1,39 @@
+package com.devfloor.hongit.core.bookmarkboard.domain
+
+import com.devfloor.hongit.core.board.domain.Board
+import com.devfloor.hongit.core.common.domain.BaseEntity
+import com.devfloor.hongit.core.user.domain.User
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.Index
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+@Entity
+@Table(
+    name = "board_bookmark",
```

Comment:
```suggestion
    name = "board_bookmarks",
```

테이블이라 복수형으로 가야할거 같어!!

Reply:
반영했어!